docs/doc/10-deploy/07-query/20-query-metrics.md

```diff
@@ -21,32 +21,57 @@ MySQL> DESC system.metrics;
-| session_connect_numbers              | The number of sessions have been connected since the server was started. The kind and labels are same as session_close_numbers.                                                                    | counter |
-| cluster_discovered_node_gauge        | The number of nodes discovered in the current cluster. The kind is gauge. The lables contain tenant_id, cluster_id, flight_address and local_id(a inner cluster unique id).                        |  gauge  |
-| parser_parse_usedtime_sum            | The sum of sql parse used time. The kind is untyped.                                                                                                                                               | untyped |
-| parser_parse_usedtime_count          | The count of sql parse used. The kind is same as parser_parse_usedtime_sum.                                                                                                                        | untyped |
-| interpreter_usedtime_sum             | The sum of sql interpreter used time. The kind is untyped.                                                                                                                                         | untyped |
-| interpreter_usedtime_count           | The count of sql interpreter used. The kind is same as interpreter_usedtime_sum.                                                                                                                   | untyped |
-| mysql_process_request_duration_sum   | The sum of MySQL interactive process request used time. The kind is untyped.                                                                                                                       | untyped |
-| mysql_process_request_duration_count | The count of MySQL interactive process request used. The kind is same as mysql_process_request_duration_sum.                                                                                       | untyped |
-| interpreter_usedtime                 | Sql interpreter used time. The kind is summary that used to track the distribution of a set of interpreter_usedtime. Value quantile contains: [0.0, 0.5, 0.9, 0.99, 0.999].                        | summary |
-| mysql_process_request_duration       | MySQL interactive process request used. The kind is summary that used to track the distribution of a set of mysql_process_request_duration. Value quantile contains: [0.0, 0.5, 0.9, 0.99, 0.999]. | summary |
-| parser_parse_usedtime                | Sql parse used time. The kind is summary that used to track the distribution of a set of parser_parse_usedtime. Value quantile contains: [0.0, 0.5, 0.9, 0.99, 0.999].                             | summary |
+:::info
+
+Metrics with `summary` kind are used to track the distribution of a set of the corresponding time consumtion.
```

Comment:
consumtion -> consumption

Reply:
Can we say "Databend uses Summary metrics to keep track of the total time spent handling various requests."?

Reply:
https://github.com/datafuselabs/databend/pull/8959